### PR TITLE
restore github workflow files

### PR DESCRIPTION
**Pull request by thimy**
I forgot to readd the `.github/workflows` folder and it seems it's blocking PRs as there's no test to be executed.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: Deploy to GitHub Pages
```
Comment (paulRbr) [inline .github/workflows/deploy.yml:1]: Do we really need this workflow file as we deploy on Netlify (and not on GitHub pages)?

Comment (thimy) [reply .github/workflows/deploy.yml:1]: I just moved the files we had on docusaurus there sorry 😅 
I thought this was what was preventing me from merging this PR https://github.com/bump-sh/docs/pull/101 but now I'm not so sure.
Should I remove this file completely?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: yarn
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Test build website
+        run: yarn build
```
Comment (paulRbr) [inline .github/workflows/test.yml:22]: Is this how we run middletown? I think it's the build task of docusaurus isn't it?

I'd recommend using these steps:
```yaml
    steps:
      - uses: actions/checkout@v3
      - uses: ruby/setup-ruby@v1
        with:
          bundler-cache: true
      - name: Test build website
        run: bin/bridgetown build
```

Comment (thimy) [reply .github/workflows/test.yml]: I didn't give it much of a thought to be honest, what would I do without you 👀

**Review by paulRbr — APPROVED** [review]
LGTM thanks!

**Pull request merged**